drivers/custom_imports_boston_community_of_origin/extensions/grda_warehouse/hud/enrollment_extension.rb

```diff
@@ -0,0 +1,62 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module CustomImportsBostonCommunityOfOrigin::GrdaWarehouse::Hud
+  module EnrollmentExtension
+    extend ActiveSupport::Concern
+
+    included do
+      has_one :client_location, class_name: 'ClientLocationHistory::Location', as: :source
+
+      scope :with_location_data, -> do
+        where.not(LastPermanentZIP: nil)
+      end
```

Comment:
Is this correct?  It feels like it should be looking to the new table to determine if there is location data (in addition to LastPermanentZIP)

Reply:
This part is to process location information that is in an Enrollment. Since `LastPermanentZIP` is being retired, it is probably short lived.